### PR TITLE
Set default UNL theme font to Gotham

### PR DESCRIPTION
**Pull request by skoolbus39**
Replace Mercury (serif) with Gotham (sans serif) as the default font for the UNL 5.2 theme.

Replaces #1428

### CODE REVIEW

File: wdn/templates_5.2/scss/critical.scss [hunk 1]
```diff
@@ -1744,12 +1741,12 @@ a.unl-cream:link {
...
 // }
-// .unl-font-serif {
-//     font-family: Mercury SSm A,Mercury SSm B,Georgia,serif!important;
-// }
-.unl-font-sans {
-  font-family: Gotham SSm A,Gotham SSm B,Verdana,sans-serif!important;
+.unl-font-serif {
+  font-family: Mercury SSm A,Mercury SSm B,Georgia,serif!important;
 }
+// .unl-font-sans {
```
Comment (jsturek) [inline wdn/templates_5.2/scss/critical.scss:1747]: Do you want it in commented out or just remove?

**Comment by jsturek** [comment]
Files to double-check to verify `unl-font-sans` references within `wdn/templates_5.2`:
* includes/global/cta-header-1.html line 1
* includes/global/apply-header-1.html line 3
* includes/global/cta-nav1.html line 1
* includes/global/give-header-1.html line 3
* includes/global/visit-header-1.html line 3
* examples/indes.shtml line 131
* js-src/events-band.js lines 54,56,63,70
* js-src/idm.babel.js lines 374, 399
* js-src/plugins/ui/css/jquery-ui-wdn.scss lines 54, 128, 160
* js-src/unlalert.babel.js line 174
* scss/deprecated/deprecated.figures.scss line 8
* scss/deprecated/deprecated.forms.scss lines 24, 48, 131
* scss/deprecated/deprecated.hero.scss line 43
* scss/deprecated/deprecated.images.scss line 24
* scss/deprecated/deprecated.infographics.scss line 4
* scss/deprecated/deprecated.pagination.scss line 7
* scss/deprecated/deprecated.promo-image.scss line 40
* scss/deprecated/deprecated.tables.scss line 9
* scss/deprecated/deprecated.text.scss lines 8,13,21,25,32,42,53
* scss/elements/_elements.details-summary.scss line 17
* scss/elements/_elements.headings.scss line 12
* scss/generic/_generic.base.scss line 7
* scss/mixins/_mixins.fonts.scss line 13
* scss/utilities/_utilities.fonts.scss lines 13,14

**Comment by skoolbus39** [comment]
The `font-sans` mixin (mixins.fonts.scss, line 13) is still used by generic.base.scss (line 7) to set the default font (separate from the critical CSS). I think that while it will be likely be rarely used, we should keep around the `.unl-font-sans` utility just in case.

File: wdn/templates_5.2/scss/critical.scss (hunk 1 above)
Comment (skoolbus39) [reply wdn/templates_5.2/scss/critical.scss:1747]: The critical CSS was initially built by copying/pasting some common styles, commenting out all of them, then selectively uncommenting out the most important styles until we reached ~12–13kb. We could revisit the Sass file and clean it up, but if we do, let's do it in a separate PR.

**Comment by jsturek** [comment]
I missed js-src/events.js lines 54,56,63,70

**Comment by jsturek** [comment]
Looks good.

**Pull request merged**